morebits.js

```diff
@@ -3951,8 +3951,11 @@ Morebits.batchOperation = function(currentAction) {
 
 		var total = ctx.pageList.length;
 		if (!total) {
-			ctx.statusElement.info('nothing to do');
+			ctx.statusElement.info('no pages specified');
 			ctx.running = false;
+			if (ctx.postFinish) {
+				ctx.postFinish();
```

Comment:
I think this is better than modifying `fnDoneOne` or something, if only for simplicity's sake, but even `workerFailure` calls `fnDoneOne`.  It's a weird construct there overall.